Keymaps/Default (Linux).sublime-keymap

```diff
@@ -54,8 +54,8 @@
     // Format Selection
     // {"keys": ["UNBOUND"], "command": "lsp_format_document_range", "context": [{"key": "setting.lsp_active"}]},
 
-    // Document Symbols
-    // {"keys": ["UNBOUND"], "command": "lsp_document_symbols", "context": [{"key": "setting.lsp_active"}]},
+    // Document Symbols (a replacement for ST's "Goto Symbol")
+    // {"keys": ["ctrl+r"], "command": "lsp_document_symbols", "context": [{"key": "setting.lsp_active"}]},
```

Comment:
`ctrl+r` is `next_bookmark` though.

Reply:
I did some research. It is `super+r` on macOS, `ctrl+r` on Linux and Windows.